src/containers/hero/hero.tsx

```diff
@@ -1,15 +1,39 @@
-import Button from '@ui/button';
+// * Import React Hooks
```

Comment:
no need of this comments. They make the reading harder and they're self explanatory